modules/openapi-generator/src/main/java/org/openapitools/codegen/InlineModelResolver.java

```diff
@@ -621,6 +621,9 @@ private Schema modelFromProperty(OpenAPI openAPI, Schema object, String path) {
         XML xml = object.getXml();
         Map<String, Schema> properties = object.getProperties();
         Schema model = new Schema();
+        if (object.getType() != null) {
```

Comment:
I've figured out Java debugger finally and found that InlineModelResolver strips data type here.

Reply:
@wing328 Yeas, I think so. It's not so critical as `type`, but it's weird that `InlineModelResolver` can cut off that field completely. I'll test it today, maybe `format` isn't missing.